src/config/config.h

```diff
@@ -28,6 +28,8 @@ typedef enum { RAI_DEVICE_CPU = 0, RAI_DEVICE_GPU = 1 } RAI_Device;
 #define REDISAI_INFOMSG_MODEL_EXECUTION_TIMEOUT "Setting MODEL_EXECUTION_TIMEOUT parameter to"
 #define REDISAI_INFOMSG_BACKEND_MEMORY_LIMIT    "Setting BACKEND_MEMORY_LIMIT parameter to"
 
+#define REDISAI_DEFAULT_MODEL_CHUNK_SIZE (511 * 1024 * 1024)
```

Comment:
why the addition?

Reply:
To use this constant in `Config_SetModelChunkSize`